src/main/java/mate/academy/service/MovieServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.service;
+
+import mate.academy.dao.MovieDao;
+import mate.academy.exeption.DataProcessingException;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Movie;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+    @Inject
+    private MovieDao movieDao;
+
+    @Override
+    public Movie add(Movie movie) {
+        return movieDao.add(movie);
+    }
+
+    @Override
+    public Movie get(Long id) {
+        return movieDao.get(id).orElseThrow(() ->
+                new DataProcessingException("no such object was found with id:" + id));
```

Comment:
u [should](https://github.com/mate-academy/hibernate-configuration-hw#:~:text=create%20your%20custom%20unchecked%20DataProcessingException%20and%20throw%20it%20in%20the%20catch%20block%20on%20dao%20layer) use this exception in dao layer, replace it and delete unnecessary constructor from ur exception